src/core/lib/surface/legacy_channel.cc

```diff
@@ -93,8 +93,11 @@ absl::StatusOr<OrphanablePtr<Channel>> LegacyChannel::Create(
     *(*r)->stats_plugin_group =
         GlobalStatsPluginRegistry::GetStatsPluginsForServer(args);
   } else {
-    StatsPlugin::ChannelScope scope(
-        target, args.GetOwnedString(GRPC_ARG_DEFAULT_AUTHORITY).value_or(""));
+    experimental::StatsPluginChannelScope scope(
+        target, args.GetOwnedString(GRPC_ARG_DEFAULT_AUTHORITY)
+                    .value_or(CoreConfiguration::Get()
```

Comment:
I've done some digging to remind myself of the flow here (details originally came from https://github.com/grpc/grpc/pull/14615#issuecomment-371273483), and I've convinced myself that this approach makes sense.  There's some more cleanup that will be needed here, but I'll take care of that in the v3 stack.

Reply:
Thanks!